content/en/continuous_integration/tests/setup/javascript.md

```diff
@@ -144,11 +144,67 @@ NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app yarn t
 
 ### Adding custom tags to tests
 
-Custom tags are not supported for Playwright.
+You can add custom tags to your tests by using the [custom annotations API from Playwright][1]:
+
+```javascript
+test('user profile', async ({ page }) => {
+  test.info().annotations.push({
+    type: 'DD_TAGS[test.memory.usage]', // DD_TAGS is mandatory and case sensitive
+    description: 'low',
+  });
+  test.info().annotations.push({
+    type: 'DD_TAGS[test.task.id]',
+    description: '41123',
+  });
+  // ...
+});
+
+test('landing page', async ({ page }) => {
+  test.info().annotations.push({
+    type: 'DD_TAGS[test.cpu.usage]',
+    description: 'high',
+  });
+  // ...
+});
+```
+
+The format of the annotations is the following
+```json
+{
+  "type": "DD_TAGS[$TAG_NAME]",
+  "description": "$TAG_VALUE"
+}
+```
+where `$TAG_NAME` and `$TAG_VALUE` are **strings** representing tag name and value respectively.
 
 ### Adding custom metrics to tests
 
-Custom metrics are not supported for Playwright.
+Custom metrics also use custom annotations:
+
+```javascript
+test('user profile', async ({ page }) => {
+  test.info().annotations.push({
+    type: 'DD_TAGS[test.memory.allocations]', // DD_TAGS is mandatory and case sensitive
+    description: 16, // this is a number now
+  });
+});
+```
+
+The format of the annotations is the following
+```json
+{
+  "type": "DD_TAGS[$TAG_NAME]",
+  "description": $TAG_VALUE
+}
+```
+where `$TAG_NAME` is a **string** representing the tag name and `$TAG_VALUE` is a **number** representing the tag value. **Note**: `description` in annotations are [typed as strings][2]. Numbers will work, but you may need to disable the typing error with `// @ts-expect-error`.
```

Comment:
~~~suggestion
The format of the annotations is the following, where `$TAG_NAME` is a *string* representing the tag name and `$TAG_VALUE` is a *number* representing the tag value:

```json
{
  "type": "DD_TAGS[$TAG_NAME]",
  "description": $TAG_VALUE
}
```
**Note**: `description` values in annotations are [typed as strings][2]. Numbers also work, but you may need to disable the typing error with `// @ts-expect-error`.
~~~